src/sentry/static/sentry/app/views/organizationDiscover/result/index.jsx

```diff
@@ -109,6 +109,13 @@ export default class Result extends React.Component {
     return <Note>{t(`Displaying up to ${NUMBER_OF_SERIES_BY_DAY} results`)}</Note>;
   }
 
+  truncate(string) {
+    if (string.length > 80) {
```

Comment:
I would make a local constant `TRUNCATE_LENGTH = 80` since we use this in multiple places in this component